examples/data-objects/table-document/README.md

```diff
@@ -1,5 +1,14 @@
 # @fluid-example/table-document
 
+<!-- AUTO-GENERATED-CONTENT:START (LIBRARY_README_HEADER) -->
```

Comment:
TODO: this is a case that was not previously considered. We do a have a few example _libraries_ in addition to example _apps_. We will want to update the library README header template to include a notice for example libraries. Ideally, these shouldn't be published. But this package, in particular, is (see the note starting on line 30 for context). Anyone who sees this package on npm or in our github should be confronted with a notice about the nature of this libarary.

Will address in a follow-up PR. Leaving the template here for now so it will auto-update with that PR.

Reply:
https://github.com/microsoft/FluidFramework/pull/22015